Assets/Scripts/DataModels/Pilots/PilotAccessors.cs

```diff
@@ -35,10 +35,6 @@ public bool IsAssignedToShip
     {
         get => saveData.isAssignedToShip; set { saveData.isAssignedToShip = value; }
     }
-
-    public Ship Ship 
-    {
-        get => ShipsManager.Instance.Ships.FirstOrDefault(s => s.Pilot == this); 

```

Comment:
Scrapped this and went back to the old, circular reference. Don't have to look up the ship each time, and don't need one-to-one between ships and pilots (which atm is annoying lol, and throwing null ref).